cpu/samd5x/Kconfig

```diff
@@ -5,4 +5,34 @@
 # directory for more details.
 #
 
+config CPU_FAM_SAMD5X
+    bool
+    select CPU_COMMON_SAM0
+    select CPU_CORE_CORTEX_M4F
+    select HAS_BACKUP_RAM
+    select HAS_CORTEXM_MPU
+    select HAS_CPU_SAMD5X
+    select HAS_PERIPH_HWRNG
+
+## CPU Models
+config CPU_MODEL_SAME54P20A
+    bool
+    select CPU_FAM_SAMD5X
+
+## Declaration of specific features
+config HAS_CPU_SAMD5X
+    bool
+    help
+        Indicates that a 'samd5x' cpu is being used.
+
+## CPU common symbols
+config CPU_FAM
+    default "samd5x" if CPU_FAM_SAMD5X
+
+config CPU_MODEL
+    default "same54p20a" if CPU_MODEL_SAME54P20A
```

Comment:
Just noticed that now, but do we have to maintain a list of all MCUs here again?
I was quite happy that we would just have to set  `CPU_MODEL` and the right vendor file and parameters get selected automatically just based on that.

Reply:
Well, when we switch to Kconfig `CPU_MODEL` will no longer be set in the Makefiles, we will be able to use the variables `CONFIG_CPU_MODEL_<model>` (boolean) and `CONFIG_CPU_MODEL` (string). This last one will be the same as `CPU_MODEL` currently.